### PR TITLE
Added packages for the Linter setting for TL & Jest-DOM | Cleanup and Refactoring

### PR DESCRIPTION
**Pull request by JacobMGEvans**
- Found a stray comment for debugging, removed
- Found ESLint issue with Airbnb config -- See airbnb/javascript#2331
Attempting to fix by manually adding a rule, this did not work, however,
Based on suggestions from the issue adding the carrot for `"eslint-plugin-import": "^2.22.1", "eslint-config-airbnb": "^18.0.1",  `in the `package.json` helps
with the import infinity issue in the Airbnb extends

### CODE REVIEW

**Comment by JacobMGEvans** [comment]
- Cleanup of comments and unused variables
- Moved imports in order based on ESLint rule 

Screen Shot 2020-12-06 at 9 18 34 PM

**Comment by JacobMGEvans** [comment]
- Modified an old test component that was also making assertions but was no longer in use to be its own test
**Overall coverage: 93.16%** 

Screen Shot 2020-12-06 at 10 17 34 PM

**Review by merodiro — APPROVED** [review]
LGTM

**Review by juhanakristian — APPROVED** [review]
lgtm

File: test/useTranslation.spec.js [hunk 1]
```diff
@@ -53,14 +53,15 @@ describe('useTranslation', () => {
...
+      const { result } = renderHook(() =>
+        useTranslation(['other', 'translation'], { i18n: i18nInstance }),
+      );
+      const { t, i18n } = result.current;
       expect(typeof t).toBe('function');
       expect(i18n).toEqual(i18nInstance);
-
-      return <div>{t('key1')}</div>;
-    }
+      expect(<div>{t('key1')}</div>).toEqual(<div>key1</div>);
```
Comment (balavishnuvj) [inline test/useTranslation.spec.js]: does it have to wrapped in dev? can't we check the output `expect(t('key1')).toEqual('key1');`

Comment (JacobMGEvans) [reply test/useTranslation.spec.js]: Jest said that was the `object` it was expecting when I tried other assertions. 🤷🏻‍♂️

Comment (balavishnuvj) [reply test/useTranslation.spec.js]: weird, we had done similar in other tests. See if that helps. If not this seems okay.

Comment (JacobMGEvans) [reply test/useTranslation.spec.js]: No good. 
Screen Shot 2020-12-07 at 9 38 42 AM

**Review by tigerabrodi — APPROVED** [review]
Looking fire :metal: :fire: :heart_eyes: 

I also did test it locally :tada:

File: test/useTranslation.spec.js (hunk 1 above)
Comment (JacobMGEvans) [reply test/useTranslation.spec.js]: Oh oops I forgot to remove the other Div lol

**Comment by tigerabrodi** [comment]
@JacobMGEvans Thanks :smiling_face_with_three_hearts:

**Pull request merged**